scrapers/Milovana.yml

```diff
@@ -0,0 +1,27 @@
+name: Milovana
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - milovana.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //wearenotreallyinterestedinscrapingatitleherebutbuildfailsifwedontmakeanattempt
```

Comment:
You mention that the title is not needed. Since we scrape the post is there any reason not to scrape it though?
something like 
```yaml
      Title:
        selector: //h3[@class="first"][contains(translate(., 'AELRS', 'aelrs'),'release')]
        postProcess:
          - replace:
              - regex: (?i)^\[\s*Release\s*\]\s*
                with: ""
```
?

Reply:
Hm. You might be right, an attempt for a title from the topic could perhaps at least be made. My reason for not including it is that it's potentially a low quality source. There's no enforced standard for a release topic even tho the majority seem to follow the [Release] pattern. Here are examples of topics from the current first page that would need manual intervention:
- [2/3 RELEASED] `TITLE` [Updated: 10/12/22]
- [RELEASE] `TITLE` / (+Poll on next production)
- [RELEASE] `TITLE` UPDATE: AVAILABILITY NOW LIMITED

But, seeing that most would actually successfully provide an acceptable title it might be worth a shot. But then I wouldn't trust this scraper to run in auto mode. And perhaps that alone, the risk of potentially worsening the users data is reason enough to not include the title? But on the other hand I think it's possible to tell stash to not overwrite the title when running the scraper, but one might forget (or not know the importance) to set that option though......

Reply:
From my understanding this is a `best effort` scraper, match whatever you can so that i dont have to go through the whole post to search for data. There can be cases where even the image won't match or the date is not the release date (maybe a separate actual release date is mentioned in the post text and so different from the @datetime). So i dont know what do you mean auto (it is a byURL scraper so identify or tagger will not work with it) but even without the title i would need to double check the post, maybe even get some text as description manually

Anyway given that this is a very specialized scraper and the users that will use it probably know what they are doing i will merge as is

Reply:
I've created a new PR with title (regex simply removes any brackets/parenthesis + content.
Also added ability to search the forum.
https://github.com/stashapp/CommunityScrapers/pull/1175